src/libsyntax_ext/lib.rs

```diff
@@ -24,6 +24,7 @@
 #![feature(str_char)]
 
 extern crate fmt_macros;
+#[macro_use] extern crate log;
```

Comment:
I think this needs corresponding dependency edges in both Makefiles and rustbuild.